applications/inquiry_proposal_applicants/FoxGaming.md

```diff
@@ -0,0 +1,84 @@
+##### ***EGF Proposal Form (FoxGaming)***
+
+## ***Overview***
+
+**The project name**: Fox Gaming
+
+**Summary of your project and how it would help our ecosystem:** NFT-Marketplace is a must have every blockchain should have to attract new investors/users. Having it will increase transactions, users and  fee´s on the fx blockchain. 
+
+**Team Leadername:** Loil -Fox Gaming
+
+**Payment address:**  fx1780uflqpght7etzq8auv0ram6rwy3wd6dasex9 f(x)Core (Function X Mainnet), $FX
+
+**The level of amount you are requesting for? Level 1,2, 3, 4:** Level 3 50,000 USD in FX tokens
+
+**Distribution schematic:**
+
+1. 25,000 USD after the proposal passes (hiring known NFT Artists to kick start the marketplace)
+2. 25,000 USD split into five tranches each 5,000USD. After significant development and improvement as:
+
+(nft launchpad, marketplace design 2.0 , daily user milestone perhaps 100-1000 daily users and so on)
+
+We apply to the three-person council (fx foundation) to release the future tranchesb after we can prove the improvements.
+
+**Category your project belongs to:** NFT Marketplace
+
+
+## ***Project Description***
+
+See all hands meeting and forum:
+
+https://www.youtube.com/watch?v=CLCGdvxwM0E
```

Comment:
could you add a link to the timestamp of the video where you've presented?

Reply:
done